docs/source/app-dev/authorization.rst

```diff
@@ -1,16 +1,16 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-.. _authentication:
+.. _authorization:
 
-Authentication
-##############
+Authorization
+#############
 
 When developing DAML applications using SDK tools,
-your local setup will most likely not use any authentication -
+your local setup will most likely not verify any authorization -
 by default, any valid ledger API request will be accepted by the sandbox.
```

Comment:
```suggestion
by default. Any valid ledger API request will be accepted by the sandbox.
```

Reply:
Note the `-` before. This is a single sentence that states the default behavior, which is to accept any valid request without verifying its authorization.